target/linux/mpc85xx/p1020/config-default

```diff
@@ -45,6 +45,7 @@ CONFIG_SWCONFIG_B53_PHY_DRIVER=y
 # CONFIG_SWCONFIG_B53_PHY_FIXUP is not set
 # CONFIG_SWCONFIG_B53_SPI_DRIVER is not set
 # CONFIG_SWCONFIG_B53_SRAB_DRIVER is not set
+CONFIG_TARGET_ROOTFS_EXT4FS=y
```

Comment:
Hm, this is an OpenWrt Config symbol in a Linux Kernel config? I don't think you can force it that way.
What you can do is to specify the FILESYSTEMS variable in the image/p1020.mk.

However, from what I can see this setup with ext4 looks odd. Wouldn't it be possible to do a setup like the MyBook Live does? It too needs a (special v1) ext2 boot partition for the linux image + dtb since the uboot can't boot of anything else. The main rootfs is configurable as either ext4 or that squashfs + f2fs (like the RPI). This should also cut down on your special sysupgrade script. Or is there something that prevents doing this?

Reply:
why are these SWCONFIG symbols here?

Reply:
They shouldn't be there, it was a mistake